src/Workspaces/Core/Portable/Recommendations/AbstractRecommendationServiceRunner.cs

```diff
@@ -282,6 +282,50 @@ protected ImmutableArray<ISymbol> GetSymbolsForNamespaceDeclarationNameContext<T
             return symbols;
         }
 
+        protected ImmutableArray<ISymbol> GetSymbolsForEnumBaseList(INamespaceOrTypeSymbol container)
+        {
+            var semanticModel = _context.SemanticModel;
+            var systemNamespace = container is not (null or INamespaceSymbol { IsGlobalNamespace: true })
+                ? null
+                 : semanticModel.LookupNamespacesAndTypes(_context.Position, semanticModel.Compilation.GlobalNamespace, nameof(System))
+                     .OfType<INamespaceSymbol>().FirstOrDefault();
+
+            using var _ = ArrayBuilder<ISymbol>.GetInstance(out var builder);
+
+            if (systemNamespace is not null)
+            {
+                builder.Add(systemNamespace);
+
+                var aliases = semanticModel.LookupSymbols(_context.Position, container).OfType<IAliasSymbol>().Where(a => systemNamespace.Equals(a.Target));
+                builder.AddRange(aliases);
+            }
+
+            AddSpecialTypeSymbolAndItsAliases(nameof(Byte), SpecialType.System_Byte);
+            AddSpecialTypeSymbolAndItsAliases(nameof(SByte), SpecialType.System_SByte);
+            AddSpecialTypeSymbolAndItsAliases(nameof(Int16), SpecialType.System_Int16);
+            AddSpecialTypeSymbolAndItsAliases(nameof(UInt16), SpecialType.System_UInt16);
+            AddSpecialTypeSymbolAndItsAliases(nameof(Int32), SpecialType.System_Int32);
+            AddSpecialTypeSymbolAndItsAliases(nameof(UInt32), SpecialType.System_UInt32);
+            AddSpecialTypeSymbolAndItsAliases(nameof(Int64), SpecialType.System_Int64);
+            AddSpecialTypeSymbolAndItsAliases(nameof(UInt64), SpecialType.System_UInt64);
+
+            return builder.ToImmutableAndClear();
+
+            void AddSpecialTypeSymbolAndItsAliases(string name, SpecialType specialType)
+            {
+                var specialTypeSymbol = _context.SemanticModel
+                    .LookupNamespacesAndTypes(_context.Position, container, name)
+                    .SingleOrDefault(s => s is INamedTypeSymbol namedType && namedType.SpecialType == specialType);
```

Comment:
```suggestion
                    .FirstOrDefault(s => s is INamedTypeSymbol namedType && namedType.SpecialType == specialType);
```

.Single can throw.  Unlikely, but i don't ever want to have to deal with a bug on that.